### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by lyhnsn** [MIV-XJTU/ARTrack#70]
Thank you for your excellent work.

There may be a bug, in "lib/train/actors/artrack_seq.py", line 232, 

`loss, status = self.compute_losses(out_dict, data)`

but it should be "self.compute_sequence_losses()"? As there is no `self.compute_losses` in this file but in "lib/train/actors/artrack.py".

Furthermore, the input of these methods are sightly different.

Looking for you reply.

**Comment by AlexDotHam** [comment]
The bug you find is interesting, because i do not use the ARTrackSeq's forward function, in another way, in seq_trainer, we use the computer_sequence_loss instead of forward (or call function). Maybe i leave this just because i transfer the sequence training code from one-stage training. Moreover, thanks for your reply, i will update it when i finish the paper recently.
